src/pool/builder.rs

```diff
@@ -5,15 +5,19 @@ use crate::pool::worker::WorkerThread;
 use crate::pool::{CloneRunnerBuilder, Local, Remote, Runner, RunnerBuilder, ThreadPool};
 use crate::queue::{self, multilevel, LocalQueue, QueueType, TaskCell};
 use crate::task::{callback, future};
-use std::sync::{Arc, Mutex};
+use std::sync::{
+    atomic::{AtomicUsize, Ordering},
+    Arc, Mutex,
+};
 use std::thread;
 use std::time::Duration;
 
 /// Configuration for schedule algorithm.
-#[derive(Clone)]
 pub struct SchedConfig {
     /// The maximum number of running threads at the same time.
     pub max_thread_count: usize,
+    /// The core number of running threads at the same time.
+    pub core_thread_count: AtomicUsize,
```

Comment:
Can you add some comment to explain why `AtomicUsize` is used?

Reply:
Fixed, thanks!